### PR TITLE
Disable spotify installation in install scripts

### PR DESCRIPTION
**Pull request by Groovylein**
I focused on buster installation.
The stretch installation changes were only a quick fix.

Not tested yet.

Have no time for that right now. Can someone test it?

### CODE REVIEW

File: scripts/installscripts/buster-install-default-with-autohotspot.sh [hunk 1]
```diff
@@ -454,60 +454,19 @@ config_spotify() {
...
-            SPOTinstall=YES
-            clear
-            echo "#####################################################
-#
-# CREDENTIALS for Spotify
-#
-# Requires Spotify username, password, client_id and client_secret
-# to get connection to Spotify.
+# IMPORTANT: Spotify disabled access to libspotify.
+#            Therefore no spotify integration possible.
```
Comment (s-martin) [inline scripts/installscripts/buster-install-default-with-autohotspot.sh:458]: Suggestion: “…no spotify integration possible at the moment.”

File: scripts/installscripts/buster-install-default.sh [hunk 2]
```diff
@@ -454,60 +454,18 @@ config_spotify() {
...
-            SPOTinstall=YES
-            clear
-            echo "#####################################################
-#
-# CREDENTIALS for Spotify
-#
-# Requires Spotify username, password, client_id and client_secret
-# to get connection to Spotify.
+# IMPORTANT: Spotify disabled access to libspotify.
+#            Therefore no spotify integration possible.
```
Comment (s-martin) [inline scripts/installscripts/buster-install-default.sh:458]: See above

File: scripts/installscripts/stretch-install-default.sh [hunk 3]
```diff
@@ -296,70 +296,20 @@ clear
...
-        clear
-        echo "This was a great decision! Mopidy will be set up."
-        echo "#####################################################
-#
-# CONFIGURE MOPIDY
-#
-# Requires spotify username, password, client_id and client_secret
-# to get connection to Spotify.
+# IMPORTANT: Spotify disabled access to libspotify.
+#            Therefore no spotify integration possible.
```
Comment (s-martin) [inline scripts/installscripts/stretch-install-default.sh:302]: See above

File: scripts/installscripts/stretch-install-default.sh [hunk 4]
```diff
@@ -296,70 +296,20 @@ clear
```
Comment (s-martin) [inline scripts/installscripts/stretch-install-default.sh:296]: We could delete the stretch file. AS far as I remember the installation on stretch doesn’t work anymore.

File: scripts/installscripts/buster-install-default.sh [hunk 5]
```diff
@@ -792,7 +750,6 @@ install_main() {
```
Comment (s-martin) [inline scripts/installscripts/buster-install-default.sh:750]: mopidy is also not necessary

**Review by s-martin — APPROVED** [review]
Question is, if we want to remove the Spotify stuff or just comment it out?

**Comment by s-martin** [comment]
As #2064 is around the corner, I think we could close this PR, @Groovylein ?

**Comment by AlvinSchiller** [comment]
Obsolete with implementation of #2315

**Pull request closed without merging**